src/test/java/bio/terra/service/dataset/ValidateAssetUnitTest.java

```diff
@@ -2,229 +2,188 @@
         List.of(
-            DatasetFixtures.generateAssetTable(tableName, List.of(invalidColumn, invalidColumn2))));
-    try {
-      dataset.validateDatasetAssetSpecification(assetModel);
-    } catch (InvalidAssetException ex) {
-      assertThat(
-          "At least one validation error caught for asset",
-          ex.getMessage(),
-          containsString("Invalid asset create request. See causes list for details."));
-      assertThat("3 errors found", ex.getCauses().size(), equalTo(3));
-      assertThat(
-          "Validation of asset model should fail on invalid root table",
-          ex.getCauses().get(0),
-          containsString("Root table " + invalidRootTable + " does not exist in dataset."));
-      assertThat(
-          "Validation of asset model should fail on invalid column",
-          ex.getCauses().get(1),
-          containsString("Column " + invalidColumn + " does not exist in table " + tableName));
-      assertThat(
-          "Validation of asset model should fail on second invalid column",
-          ex.getCauses().get(2),
-          containsString("Column " + invalidColumn2 + " does not exist in table " + tableName));
-      throw ex;
-    }
+            DatasetFixtures.generateAssetTable(
+                TABLE_NAME, List.of(invalidColumn, invalidColumn2))));
+    InvalidAssetException ex =
+        assertThrows(
+            InvalidAssetException.class,
+            () -> dataset.validateDatasetAssetSpecification(assetModel));
+    assertThat(
+        "At least one validation error caught for asset",
+        ex.getMessage(),
+        containsString("Invalid asset create request. See causes list for details."));
+    assertThat("3 errors found", ex.getCauses(), hasSize(3));
+    assertThat(
+        "Validation of asset model should fail on invalid root table",
+        ex.getCauses().get(0),
+        containsString("Root table " + invalidRootTable + " does not exist in dataset."));
+    assertThat(
+        "Validation of asset model should fail on invalid column",
+        ex.getCauses().get(1),
+        containsString("Column " + invalidColumn + " does not exist in table " + TABLE_NAME));
+    assertThat(
+        "Validation of asset model should fail on second invalid column",
+        ex.getCauses().get(2),
+        containsString("Column " + invalidColumn2 + " does not exist in table " + TABLE_NAME));
   }
 
   private void testAssetModel(String itemChecked, String expectedErrorMessage) {
-    try {
-      dataset.validateDatasetAssetSpecification(assetModel);
-    } catch (InvalidAssetException ex) {
-      assertThat(
-          "At least one validation error caught for asset",
-          ex.getMessage(),
-          containsString("Invalid asset create request. See causes list for details."));
-      assertThat(
-          "Validation of asset model should fail on " + itemChecked,
-          ex.getCauses().get(0),
-          containsString(expectedErrorMessage));
-      throw ex;
```

Comment:
The tests here used to depend on `InvalidAssetException` being rethrown. Now that we're using `assertThrows()`, we no longer have to do this.